### DISCUSSION TITLE
Dumping Database stuck

### DISCUSSION THREAD

**Discussion by AmityTek** [spatie/laravel-backup#1735] [Q&A]
Hi,

I am using your package to dump my DB's (i have 3 DB)
Everything was working well, but now dumping database is stuck for only one ( db size ~120mb)
I can see sql file 0KB inside backup-temp/temp but nothing more.

I dont have any errors, nothing ...
Any idea how can I fix this ?

Thanks for help

**Comment by clicktodev** [comment]
Can you provide information about the environment and the database?
